src/components/HomepageFromPkg.test.tsx

```diff
@@ -15,9 +15,9 @@ describe("HomepageFromPkg", () => {
       homepage: "https://dbartholomae.github.io/jsx-readme",
       name: "test-package",
     };
-    expect(await render(<HomepageFromPkg pkg={pkg} />)).toContain(
-      "## 🏠 Homepage\n"
-    );
+    expect(
+      await render(<HomepageFromPkg pkg={pkg} title="Homepage" />)
+    ).toContain("## Homepage\n");
```

Comment:
Sorry I missed this in the first review: This should be a new test, not a chang eto the old one.

Reply:
changes done .Pls check https://github.com/dbartholomae/jsx-readme/pull/56